### DISCUSSION TITLE
Communicating with ESP32 via phone app while using Bluetooth.

### DISCUSSION THREAD

**Discussion by rbagdazian** [pschatzmann/arduino-audio-tools#951] [Q&A] [answered]
I have been using the arduino-audio-tools successfully and it is a great resource. I was wondering if it would be possible to have a dedicated phone application that could communicate with the ESP 32 for the purpose of sending commands to control the ESP32's operation while still processing incoming bluetooth audio stream from the same mobile device? This would be for the purpose of say adjusting the volume of the unit or other processing that might be going on while the bluetooth audio is being processed?
Thank you in advance!

**Comment by pschatzmann** [comment, marked as answer]
As far as I can see using AVRCP seems to be your ownly option here

**Reply by rbagdazian** [reply]
OK Thank you! Something new to research.
